tests/integration/features/bootstrap/CoreContext.php

```diff
@@ -0,0 +1,930 @@
+<?php
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use GuzzleHttp\Client;
+use GuzzleHttp\Message\ResponseInterface;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+/**
+ * Features context.
+ */
+class CoreContext implements Context, SnippetAcceptingContext {
```

Comment:
The problem is it's like a rat tail... you end up copying more and more, and then it's also harder to track changes that have been done in core